Makefile

```diff
@@ -19,7 +19,7 @@ GO_INSTALL = ./hack/go-install.sh
 TOOLS_DIR=hack/tools
 GOBIN_DIR := $(abspath $(TOOLS_DIR))
 
-GOLANGCI_LINT_VER := v1.58.1
+GOLANGCI_LINT_VER := v1.62.2
```

Comment:
If we revert, still fails linting?

Reply:
It crashes immediately after running `make lint test` locally, even without any changes done. The stack trace suggests the issue is inside golangci-lint itself. Indeed it's strange that it should crash when it must have worked before. In any case, do we want to keep the older version at all? Or we update in a separate PR?